src/ResourceManager/ApiManagement/Commands.SMAPI.Test/ScenarioTests/ApiManagementTests.ps1

```diff
@@ -175,10 +175,12 @@ function Api-ImportExportWadlTest {
         Assert-AreEqual $wadlApiId $api.ApiId
         Assert-AreEqual $path $api.Path
 
+        # commented as powershell test framework on running test in playback mode, throws 403, as the exported link of file
+        # gets expired
         # export api to pipline
-        $result = Export-AzureRmApiManagementApi -Context $context -ApiId $wadlApiId -SpecificationFormat Wadl
+        # $result = Export-AzureRmApiManagementApi -Context $context -ApiId $wadlApiId -SpecificationFormat Wadl
 
-        Assert-True {$result -like '*<doc title="Yahoo News Search">Yahoo News Search API</doc>*'}
+        # Assert-True {$result -like '*<doc title="Yahoo News Search">Yahoo News Search API</doc>*'}
```

Comment:
@solankisamir it looks like there's a lot of commented-out asserts in this file -- any reason why?

Reply:
The output of the commandlet is an Uri string with time bound token. When we rerun the test in Playback mode, the commandlet throws exception, as the Uri is no longer valid.
All the commented Commandlet are the one testing `Export-AzureRmApiManagementApi`


---
In reply to: [188128363](https://github.com/Azure/azure-powershell/pull/6189#discussion_r188128363) [](ancestors = 188128363)